engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content-hosts-helper.service.js

```diff
@@ -7,6 +7,19 @@
  */
 angular.module('Bastion.content-hosts').service('ContentHostsHelper',
     function () {
+        this.convertMemToGB = function (memoryValue) {
+            if (angular.isString(memoryValue)) {
+                memoryValue = memoryValue.toLowerCase();
+                if (_.includes(memoryValue, "gb")) {
+                    memoryValue = memoryValue.replace("gb", "").trim();
+                    return memoryValue;
```

Comment:
I tried to avoid two return statements, but couldn't figure it out and handle an int, string, and string like "4 GB" (i put them in the test cases)  and not return twice while keeping the function simple. I'll revisit it  though